### PR TITLE
find the title from second page

### PR DESCRIPTION
**Pull request by leaf-soba**
- close:https://github.com/JabRef/jabref/issues/12139
- Sorry I didn't find a proper test file with a open access license, although I test the file in the issue it works fine.

### Mandatory checks

- [x] I own the copyright of the code submitted and I licence it under the [MIT license](https://github.com/JabRef/jabref/blob/main/LICENSE)
- [ ] Change in `CHANGELOG.md` described in a way that is understandable for the average user (if change is visible to the user)
- [x] Tests created for changes (if applicable)
- [x] Manually tested changed features in running JabRef (always required)
- [ ] Screenshots added in PR description (for UI changes)
- [ ] [Checked developer's documentation](https://devdocs.jabref.org/): Is the information available and up to date? If not, I outlined it in this pull request.
- [ ] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not, I created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, I submitted a pull request to the documentation repository.

### CODE REVIEW

File: src/main/java/org/jabref/logic/importer/fileformat/PdfContentImporter.java [hunk 1]
```diff
@@ -232,7 +232,7 @@ public TitleExtractorByFontSize() {
 
         public String getTitleFromFirstPage(PDDocument document) throws IOException {
```
Comment (koppor) [inline src/main/java/org/jabref/logic/importer/fileformat/PdfContentImporter.java:233]: The method name does not match the content of the method any more.

**Comment by koppor** [comment]
Test  PDF not OK - the title page is OK, but the subsequent PDF not

image

You can use for example following paper: `src\test\resources\pdfs\IEEE\ieee-paper.pdf` as content for pages 2 onwards.

I bought an Acrobat Professional X for 10€ some years ago. - I think, other PDF tools such as PDFSam also work to create a PDF.

**Review by koppor — CHANGES_REQUESTED** [review]
> I love consistency - and the title page should somehow match the PDF attached. You can also create a new PDF based on `src\test\resources\pdfs\IEEE`)

The current version has following data:

Page 1:

- Two Authors: "Test Author" asnd "Author Test"
- Titel: "A Test Title."

Page 2:

- One author: "First Author"
- Title: "JabRef Example for Reference Parsing"

image

Please make this consistent. Make the first page matchting the second page. Change "Test Author, Author Test" to "First Author". Change "A Test Title." to "JabRef Example for Reference Parsing"

**Review by leaf-soba — COMMENTED** [review]
fix the pdf file with consistent cover

**Review by koppor — APPROVED** [review]

**Comment by koppor** [comment]
> I think this needed a changelog entry?

Good point! Can you create one? 😅

**Pull request merged**